src/core/lib/event_engine/posix_engine/timer_manager.cc

```diff
@@ -35,7 +35,7 @@
 #include "src/core/lib/gpr/tls.h"
 #include "src/core/lib/gprpp/thd.h"
 
-static GPR_THREAD_LOCAL(bool) g_timer_thread = false;
+static GPR_THREAD_LOCAL(bool) g_timer_thread;
```

Comment:
is this default-initialized?

Reply:
Statics are always zero initialized

Reply:
hmm looks like gpr thread local variables no longer need initialization